src/Blockcore/Consensus/ScriptAddressReader.cs

```diff
@@ -27,14 +27,16 @@ public string GetAddressFromScriptPubKey(Network network, Script script)
                 case TxOutType.TX_SCRIPTHASH:
                     destinationAddress = script.GetDestinationAddress(network).ToString();
                     break;
+                case TxOutType.TX_SEGWIT:
+                    destinationAddress = script.GetDestinationAddress(network).ToString();
```

Comment:
will this actually be a segwit address? or do we need to do some more conversions?
I will push for now we can fix later.